pom.xml

```diff
@@ -1552,7 +1552,8 @@
 
     <!-- NOTE: When updating GeoTools, check which version
     of Postgres is used and update pg.version if needed. -->
-    <geotools.version>23.0</geotools.version>
+    <geotools.version>27.0</geotools.version>
```

Comment:
do we need to check which version of postgresql is used (as noted in the above comment)